### PR TITLE
feat(tooling): init

### PR DESCRIPTION
**Pull request by Laudenlaruto**

### CODE REVIEW

**Comment by spoukke** [comment]
Should we talk about `tfautomv` in this tooling section?

**Comment by ChapChap** [comment]
Don't we recommend `terraform_docs` ? `terrascan` ?

File: README.md [hunk 1]
```diff
@@ -94,7 +94,8 @@ You’ll find below details of the standards to follow when working with Terrafo
 
 ### 🛠️ Tooling
 
-- [Pre-commits](terraform/pre-commits.md)
+- [Useful tooling for Terragrunt/Terraform](tooling/tooling.md)
+- [Terraform Pre-commits](terraform/pre-commits.md)
```
Comment (cterence) [inline README.md:98]:
```suggestion
- [Terraform pre-commits](terraform/pre-commits.md)
```

File: tooling/tflint.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Tflint
+
+This page goal is to list some useful information about [tflint](https://github.com/terraform-linters/tflint)
```
Comment (cterence) [inline tooling/tflint.md:3]:
```suggestion
The goal of this page is to list some useful information about [tflint](https://github.com/terraform-linters/tflint)
```

File: tooling/tooling.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# Useful tooling for Terragrunt/Terraform
```
Comment (cterence) [inline tooling/tooling.md:1]: Shouldn't we rename this to README.md ?

File: tooling/tooling.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  - [Default configuration](tflint.md#default-configuration)
+  - We recommend this one because it can check a wide range of cloud providers as well as Terraform code
+- [terraform fmt](https://developer.hashicorp.com/terraform/cli/commands/fmt) - Rewrites all Terraform configuration files to a canonical format
+  - Example usage : `terraform fmt -recursive -diff -write=true`
+- [terragrunt hclfmt](https://terragrunt.gruntwork.io/docs/reference/cli-options/#hclfmt) - Rewrites all Terragrunt configuration files to a canonical format
+  - Example usage : `terragrunt hclfmt`
+
+## Code security
+
+> Left shift as much as possible, security related tasks
```
Comment (cterence) [inline tooling/tooling.md:26]:
```suggestion
> Left shift security related tasks as much as possible
```

**Review by cterence — CHANGES_REQUESTED** [review]

**Review by cterence — APPROVED** [review]

**Pull request merged**